gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-info.spec.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ADMIN_USER } from '@fakers/users/users';
+import { ApisFaker } from '@gravitee/fixtures/management/ApisFaker';
+import { Visibility } from '@gravitee/management-webclient-sdk/src/lib/models';
+import faker from '@faker-js/faker';
+import apiDetails from '@pageobjects/Apis/ApiDetails';
+import { ApiImport } from '@model/api-imports';
+
+describe('API Info Page functionality', () => {
+  beforeEach(() => {
+    cy.loginInAPIM(ADMIN_USER.username, ADMIN_USER.password);
+    cy.visit('/#!/environments/default/apis/');
+    cy.url().should('include', '/apis/');
+  });
+
+  before(() => {
+    cy.log('Import (create) v2 API');
+    cy.request({
+      method: 'POST',
+      url: `${Cypress.env('managementApi')}${Cypress.env('defaultOrgEnv')}/apis/import`,
+      auth: { username: ADMIN_USER.username, password: ADMIN_USER.password },
+      body: ApisFaker.apiImport({ visibility: Visibility.PUBLIC }),
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      api = response.body;
+    });
+  });
+
+  const ApiDetails = new apiDetails();
+  const apiName = faker.commerce.productName();
+  const newName = apiName.replace(/ /g, '-');
```

Comment:
it's sad to use faker to manually have to modify the name afterwards, maybe you can go directly with `faker.lorem.words(10)` if you're not happy with what is generated.

Reply:
The reason for this newName modification is because when an API is exported, the apiName that includes spaces "A Test Api" is exported as "A-Test-Api.json" so the .replace is just replacing all ' ' (spaces) with '-' (hyphens). Does this still require changes or has that explained it? I could just change constructor name to exportFileName or something if it is confusing.

Reply:
ok, I saw you rename the variable so it's better now.